plop/backend.mjs

```
@@ -161,6 +162,42 @@ function backendFactory(plop) {
+
+  setPlopHelpers(plop)
+}
+
+
+ function setPlopHelpers(plop){
+
+  plop.setHelper('UC', function (text) {
+    // if(text && text != '')
+      return text.toUpperCase();
+
+    return text
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)